compiler/rustc_error_codes/src/error_codes/E0462.md

```diff
@@ -0,0 +1,31 @@
+Found staticlib `..` instead of rlib or dylib.
+
+Consider the following two files:
+
+`a.rs`
+```ignore (cannot-link-with-other-tests)
+#![crate_type = "staticlib"]
+
+fn foo() {}
+```
+
+`main.rs`
+```ignore (cannot-link-with-other-tests)
+extern crate a;
+
+fn main() {
+    a::foo();
+}
+```
+
+Crate `a` is compiled as a staticlib. A staticlib is a system-dependant library
+only intended for linking with non-Rust applications (C programs). Note that
+staticlibs include all upstream dependencies (`core`, `std`, other
+user dependencies, etc) which makes them significantly larger than dylibs:
+prefer staticlib for linking with C programs. Learn more about different
+`crate_type`s in  [this section of the Reference](../reference/linkage.html).
+
+This error can be fixed by:
+ * Using [Cargo](../cargo/index.html), the Rust package manager, automatically
```

Comment:
You did it correctly! \o/

Oh also, the link for cargo is invalid. Please use the full HTTP link to the website (less likely to break).

Reply:
#98227 says that relative links are better because it avoids mismatches between nightly/ stable but I'm fine either way.

Reply:
Hum... Let's keep as is then.